raiden/blockchain/events.py

```diff
@@ -169,10 +171,13 @@ def get_relevant_proxies(pyethapp_chain, node_address, registry_address):
         channel_managers.append(channel_manager)
 
         participating_channels = channel_manager.channels_by_participant(node_address)
-        netting_channels = [
-            pyethapp_chain.netting_channel(channel_address)
-            for channel_address in participating_channels
-        ]
+        netting_channels = []
+        for channel_address in participating_channels:
+            # FIXME: implement proper cleanup of self-killed channel after close+settle
+            try:
+                netting_channels.append(pyethapp_chain.netting_channel(channel_address))
+            except ValueError:
+                log.exception('Invalid netting channel:')
```

Comment:
I think this exception logging will scary users for something that is not unexpected at all, perhaps a debug level?